kentikapi/client.go

```diff
@@ -161,8 +160,8 @@ func makeConnForGRPC(c Config, hostPort string) (grpc.ClientConnInterface, error
 	)
 }
 
-func makeTLSOption(c Config) grpc.DialOption {
-	if c.DisableTLS {
+func makeTLSOption(t bool) grpc.DialOption {
```

Comment:
Please use something logner than "t", from function's signature it's not clear what it does :)

Reply:
Done